enclave_manager/avalon_enclave_manager/avalon_enclave_bridge.py

```diff
@@ -178,32 +178,45 @@ def get_enclave_basename():
 
 
 # -----------------------------------------------------------------
-def verify_enclave_info(enclave_info, mr_enclave, originator_public_key_hash):
+def verify_enclave_info(enclave_info, mr_enclave, extended_data):
     """
     Verifies enclave signup info
     - enclave_info is a JSON serialised enclave signup info
       along with IAS attestation report
     - mr_enclave is enclave measurement value
+    - extended_data - MRENCLAVE value of associated WPE
+      in case of KME Enclave and Verification key of associated KME
+      incase of WPE Enclave
     """
-    return enclave.VerifyEnclaveInfo(
-        enclave_info, mr_enclave, originator_public_key_hash)
+    return enclave.VerifyEnclaveInfo(enclave_info, mr_enclave, extended_data)
 
 
 # -----------------------------------------------------------------
-def create_signup_info(originator_public_key_hash, nonce):
+def create_signup_info(nonce, worker_mode=1, extended_data="",
```

Comment:
Can't we pass enum instead of integer for worker_mode

Reply:
Yes. I will do the changes.